docs/src/meshes/p4est_mesh.md

```diff
@@ -13,7 +13,7 @@ wrap numerical fluxes implemented only for Cartesian meshes. This simplifies
 the re-use of existing functionality for the [`TreeMesh`](@ref) but is usually
 less efficient, cf. [PR #550](https://github.com/trixi-framework/Trixi.jl/pull/550).
 
-## Construction of P4estMesh from an Abaqus file
+## Construction of 2D P4estMesh from an Abaqus file
```

Comment:
```suggestion
## Construction of a P4estMesh from an Abaqus file
```
I would remove the 2D part from this level two heading again and then introduce a level three heading
```markdown
### 2D
```
after the first paragraph that ends with `... and conforming mesh.`. Below, you could then also shorten the 3D heading to just `3D` and make it a level three heading.

Right now, the heading below reads "Short discussion on 3D" and is at the same level as the Abaqus-specific heading here, suggesting that the short discussion on 3D is unrelated to Abaqus files.

Reply:
Done